JASP-Desktop/widgets/listmodel.cpp

```diff
@@ -465,6 +479,37 @@ const QString &ListModel::name() const
 	return _listView->name();
 }
 
+const Terms &ListModel::terms(const QString &what) const
+{
+	if (what.startsWith("type="))
+	{
+		static Terms terms;
+
+		QStringList typesStr = what.right(what.length() - 5).toLower().split("|");
```

Comment:
I would be more happy with:
```
const QString typeIs = "type=";
if (what.startsWith(typeIs))
	{
		static Terms terms;

		QStringList typesStr = what.right(what.length() - typeIs.length()).toLower().split("|");
```
This way we don't get weird bugs if someone changes it (or worse, copies it and reuses it somewhere else without thinking properly)

Reply:
Done